src/backend/expungeservice/models/disposition.py

```diff
@@ -1,9 +1,14 @@
-from datetime import datetime
+from dataclasses import dataclass
+from datetime import date, datetime
+from typing import Optional
 
 
+@dataclass(eq=False)
 class Disposition:
+    date: Optional[date]
```

Comment:
I don't believe the date is optional. If the disposition is instantiated then there needs to be a date.

Reply:
I am just preserving current behavior here. By nature of the previous default argument being `date=None`, the date can be optional. It doesn't matter that the default argument is always overridden.

Reply:
Also we assume this on line 18 of this file.

Reply:
Thanks for pointing that out. I let my knowledge of the code base interfere with reading the code
.

Reply:
@KentShikama The default parameters can be removed. I think we can either put that in this PR or create a new issue. I will leave that up to you on how you want to do it. See PR #357 for more information on when the optional parameters were removed.

Reply:
I've created https://github.com/codeforpdx/recordexpungPDX/issues/625 as this is not as trivial of a change as I first thought (see the issue as to why).

Reply:
Thanks @KentShikama I thought there might still be tests that use the optional params.

Reply:
Thanks for being patient and helping me learn!